lib/matplotlib/widgets.py

```diff
@@ -1448,7 +1449,7 @@ class Cursor(AxesWidget):
 
     Other Parameters
     ----------------
-    **lineprops
+    **props
```

Comment:
Let's keep this PR limited to the selectors. These are not artist properties that control a part of a selector. Instead, these are straight-forward properties of the centeral visual element. This is rather a case for #20048, which ``**props`` would move away from further. Note that there is also a substantial difference between a regular parameter with defaults and ``**kwargs``. You cannot assign to the latter explicitly, so that its name is not public API.